app/services/notification_text_message.rb

```diff
@@ -0,0 +1,23 @@
+class NotificationTextMessage
```

Comment:
I would like you to please remove this file from this PR. I apologize, since I caused this confusion. The class is not properly named, and is about `message`s, not `notification`s. 
We'll want this file to be added once we get to message model.